inductiva/__init__.py

```diff
@@ -65,4 +68,13 @@ def _check_for_available_package_update():
         print(msg, file=sys.stderr)
 
 
+def _check_key():
+    global _checked_key
+    if not _checked_key and os.getenv("GITHUB_ACTIONS",
```

Comment:
Use `utils.format_utils.getenv_bool` to avoid a comparison with a string that is meant to be a boolean.